src/PactNet.Native/NativePactVerifier.cs

```diff
@@ -8,23 +7,13 @@ namespace PactNet.Native
     /// </summary>
     internal class NativePactVerifier : IVerifierProvider
     {
-        /// <summary>
-        /// Static constructor for <see cref="NativePactVerifier"/>
-        /// </summary>
-        static NativePactVerifier()
```

Comment:
Why removing this? Is it because of the way we will handle the logs going forward?

Reply:
It's moved into the combined interop file, then in future we can allow the user to specify the log level. I've